integreat_cms/cms/forms/push_notifications/push_notification_form.py

```diff
@@ -7,6 +15,18 @@ class PushNotificationForm(CustomModelForm):
     Form for creating and modifying push notification objects
     """
 
+    schedule_send = forms.BooleanField(label=_("Schedule sending"), required=False)
+    scheduled_send_date_day = forms.DateField(
+        label=_("Scheduled send day"),
+        required=False,
+        widget=forms.DateInput(format="%Y-%m-%d", attrs={"type": "date"}),
```

Comment:
Note that this will calculate the date once on server startup and then never update the date. Also I am not sure if that deals correctly with timezones. I have added this for now, but tbh. I don't fully understand how we deal with timezones currently.
```python
self.fields["scheduled_send_date_day"].widget.attrs["min"] = str(timezone.now().date())
```

Reply:
Timezone quickstart:
- In the db, we always store UTC
- The server time is UTC, so the management command can assume that their time is identical to the time in the db
- In the UI, we show the local time of the user. Django automatically calculates this correctly to convert this to UTC when storing it in the db

Regarding the min value: To avoid the issue with fixed date on server start, we could also calculate the min value in JS.

Reply:
> Note that this will calculate the date once on server startup and then never update the date.

Ooh, good call. I didn't think of that at all.

Reply:
> Regarding the min value: To avoid the issue with fixed date on server start, we could also calculate the min value in JS.

Do you think that is preferable over using django's provided `timezone.now()`?